packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -27,10 +27,12 @@ const { prefix } = settings;
  */
 const Card = ({
   title,
```

Comment:
It's fine if we're planning this for another pull request, but we haven't forgot about updating the props, correct?

Reply:
I think I want to put it in another PR